Build failed in Xcode 7.3.1

Hi,

I followed the "Getting Started" instructions but I'm getting errors when I build the Authenticator scheme in `Authenticator.xcworkspace`:

> <unknown>:0: error: underlying Objective-C module 'OneTimePassword' not found
> /Users/user/pathtoiosdir/Authenticator/Carthage/Checkouts/OneTimePassword/OneTimePassword/Token+URL.swift:27:8: error: no such module 'Base32'
> import Base32

What am I doing wrong?
(Xcode 7.3.1)

--- a/README.md
+++ b/README.md
@@ -19,12 +19,18 @@ Authenticator is a simple, free, and open source [two-factor authentication](htt
 
 ## Getting Started
 
-1. Check out the latest version of the project using Git.
+1. Check out the latest version of the project:
+  ```
+  git clone https://github.com/mattrubin/Authenticator.git
+  ```
 
-2. Check out the project's dependencies with `git submodule init` and `git submodule update`.
+2. Check out the project's dependencies:
+  ```
+  git submodule update --init --recursive
+  ```
 
 3. Open the `Authenticator.xcworkspace` file.
-> If you open the `.xcodeproj` instead, the app will not be able to find its dependencies.
+> If you open the `.xcodeproj` instead, the project will not be able to find its dependencies.
 
 4. Build and run the "Authenticator" scheme.
 

